client/src/app/components/stock-detail-header/stock-detail-header.component.scss

```diff
@@ -4,26 +4,36 @@
     display: flex;
     align-items: center;
     margin-bottom: 20px;
+    justify-content: space-between;
+    align-items: center;
+    
+    span { 
```

Comment:
As mentioned above, we should not use a spam, and should add a class name to it instead